pkg/experimentcalculator/experimentcalc/experiment_calculator.go

```diff
@@ -78,12 +76,18 @@ func NewExperimentCalculator(
 	logger *zap.Logger,
 ) *ExperimentCalculator {
 	registerMetrics(metrics)
-	compiledModel, err := httpStan.CompileModel(context.TODO(), stan.ModelCode())
-	if err != nil {
-		logger.Error("Failed to compile model",
-			zap.Error(err),
-		)
-		return nil
+	var compiledModel stan.ModelCompileResp
+	for {
```

Comment:
We should set a limit here. Otherwise, it could enter the infinity loop.
How frequently it could fail while compiling the model?

Reply:
Currently, we don't need to compile the model, we have already compiled it in httpstan image, means when the httpstan container is ready, we can directly use the stan model.

But httpstan container may start after experiment-calculator container.

Reply:
I understand, but this kind of implementation could lead to bugs in the future, so it's safer we set a limit in case it doesn't work as expected.

Reply:
No problem, I will set it to retry 3 times to avoid this.
In my dev container, httpstan generally delay 1~2 second after experiment-calculator is ready.

Reply:
Done!